migrations/README.md

```diff
@@ -120,7 +120,17 @@ Then, populate the migration at `./db/migrations/xxxxxxxxx_<some_message>` and m
 ```shell
 docker-compose run --rm dbmate up
 ```
+### Updating schema.sql for each major version
 
+After making changes to migrations, you should update the schema.sql files for each major version of PostgreSQL:
+
+```shell
+# First, stop any running PostgreSQL servers
+# Then from the root of supabase/postgres run:
+nix run .#dbmate-tool -- --version all
+```
+
+This will create a schema.sql file for each major version of PostgreSQL. Commit these changes to your repository and push to your branch. The test.yml workflow will verify these changes against the test matrix.
```

Comment:
This will also create the `migrations/schema-orioledb-17.sql`?

Reply:
Btw why is this needed? I had a related question on https://github.com/supabase/postgres/pull/1554/files#r2049582929.

Reply:
Absolutely it will create a schema sql for 15, 17 and orioles 17 because they are different across major versions

Reply:
It will append the version on the end

Reply:
Ok cool, then we should put it here for completion:

```suggestion
This will create a schema.sql file for each major version of PostgreSQL and OrioleDB (the files are named like `schema-<ver>`, `schema-oriole-<ver>`). Commit these changes to your repository and push to your branch. The test.yml workflow will verify these changes against the test matrix.
```

Reply:
Follow up question: why do we need these `schema-<ver>` files? The changes are redundant with the files included in `migrations/` right? Is it for visibility?

Reply:
@steve it’s re run in CIO, compared to the existing file in repo and detects if migrations are covered and committed to this repo

Reply:
@samrose So they're for ensuring migrations are idempotent?

How about clarifying those are autogenerated here in the docs? 

We need to clarify that otherwise it's not clear and seems they're duplicated code, which leads to questions: https://github.com/supabase/postgres/pull/1554#discussion_r2049582929.

Reply:
@steve-chavez ok sure. Yeah this is a common practice when dbmate tool is used. But sure we can document it the way you describe so it’s clear

Reply:
@steve-chavez have a look at this and let me know if these are good enough

https://github.com/supabase/postgres/pull/1555/commits/1e95709dccd107ead82d30b326bc282188960ff7

Reply:
Yes, that looks good now. Thanks for clarifying.